src/fides/core/config/security_settings.py

```diff
@@ -16,42 +15,71 @@
-
-
 class SecuritySettings(FidesSettings):
     """Configuration settings for Security variables."""
 
-    root_user_scopes: Optional[List[str]] = SCOPE_REGISTRY
-    subject_request_download_link_ttl_seconds: int = 432000  # 5 days
-    request_rate_limit: str = "1000/minute"
-    rate_limit_prefix: str = "fides-"
-    aes_encryption_key_length: int = 16
-    aes_gcm_nonce_length: int = 12
-    app_encryption_key: str = ""
-    drp_jwt_secret: Optional[str] = None
-    root_username: Optional[str] = None
-    root_password: Optional[str] = None
-    parent_server_username: Optional[str] = None
-    parent_server_password: Optional[str] = None
-    identity_verification_attempt_limit: int = 3  # 3 attempts
-    encoding: str = "UTF-8"
-    env: SecurityEnv = SecurityEnv.DEV
-
-    cors_origins: List[str] = []
-    oauth_root_client_id: str = ""
-    oauth_root_client_secret: str = ""
-    oauth_root_client_secret_hash: Optional[Tuple]
-    oauth_access_token_expire_minutes: int = 60 * 24 * 8
-    oauth_client_id_length_bytes = 16
-    oauth_client_secret_length_bytes = 16
+    aes_encryption_key_length: int = Field(default=16, description="TODO")
+    aes_gcm_nonce_length: int = Field(default=12, description="TODO")
+    app_encryption_key: str = Field(
+        default="", description="The key used to sign Fides API access tokens."
+    )
+    cors_origins: List[str] = Field(
+        default=[],
+        description="A list of pre-approved addresses of clients allowed to communicate with the Fides application server.",
+    )
+    drp_jwt_secret: Optional[str] = Field(default=None, description="TODO")
+    encoding: str = Field(
+        default="UTF-8", description="Text encoding to use for the application."
+    )
+    env: str = Field(
+        default="dev",
+        description="The default, `dev`, does not apply authentication to endpoints typically used by the CLI. The other option, `prod`, requires authentication for _all_ endpoints that may contain sensitive information.",
+    )
+    identity_verification_attempt_limit: int = Field(default=3, description="")
+    oauth_root_client_id: str = Field(
+        default="",
+        description="The value used to identify the Fides application root API client.",
+    )
+    oauth_root_client_secret: str = Field(
+        default="",
+        description="The secret value used to authenticate the Fides application root API client.",
+    )
+    oauth_root_client_secret_hash: Optional[Tuple] = Field(
+        default=None, description="TODO"
+    )
+    oauth_access_token_expire_minutes: int = Field(
+        default=11520,
+        description="The time in minutes for which Fides API tokens will be valid. Default value is equal to 8 days.",
+    )
+    oauth_client_id_length_bytes: int = Field(default=16, description="TODO")
+    oauth_client_secret_length_bytes: int = Field(default=16, description="TODO")
+    parent_server_password: Optional[str] = Field(default=None, description="TODO")
```

Comment:
Help

Reply:
When using a parent/child fides setup, this is the password the child will use to access the parent server

Reply:
probably @sanders41

Reply:
```suggestion
    parent_server_password: Optional[str] = Field(default=None, description="When using a parent/child Fides deployment, this password will be used by the child server to access the parent server.")
```